test/functional/test-action.js

```diff
@@ -1037,4 +1038,29 @@ describe('Core events', () => {
         }));
   });
 
+  describe('DeferredEvent', () => {
+    it('should copy the properties of an event object', () => {
+      const event = createCustomEvent(window, 'MyEvent', {foo: 'bar'});
+      const deferredEvent = new DeferredEvent(event);
+
+      for (const key in deferredEvent) {
+        if (typeof deferredEvent[key] !== 'function') {
+          expect(deferredEvent[key]).to.deep.equal(event[key]);
```

Comment:
Unfortunately Chrome gives CustomEvent instances a `path` property and when you assign it to a variable, that variable is somehow not equal to the value. using `deep` solves this. I'm not sure what else to do other than special-casing this property in the test.

<img width="379" alt="screen shot 2017-05-16 at 10 20 32 am" src="https://cloud.githubusercontent.com/assets/2363700/26119275/8c69b812-3a21-11e7-93ac-194d55335cc5.png">